Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInSignUp.kt

```diff
@@ -191,14 +187,13 @@ fun Password(
                     passwordState.enableShowErrors()
                 }
             },
-        textStyle = MaterialTheme.typography.body2,
+        textStyle = MaterialTheme.typography.bodyMedium,
         label = {
-            CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
-                Text(
-                    text = label,
-                    style = MaterialTheme.typography.body2
-                )
-            }
+            Text(
+                text = label,
+                style = MaterialTheme.typography.bodyMedium,
+                color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.6f)
```

Comment:
Ideally this should rather be using `colorScheme.onSurfaceVariant` as an equivalent to `ContentAlpha.medium`. Was this alpha value specified in the design? Same goes for others below. If not, a constant for `0.6f` would be great.

Reply:
Yeah, this was specified as "Jetsurvey/sys/light/on-surface" and opacity 60% in the design (same with the one below). Good call on the constant; I'll do that.